run/ktor-hello-world/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
We should be able to remove this file as it's covered by our root `LICENCE` file, which is [the same license](https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/LICENSE)